addons/mail/static/src/components/message/message.xml

```diff
@@ -56,31 +56,33 @@
                     <div class="o_Message_core flex-grow-1">
                         <t t-if="!messageView.isSquashed">
                             <div class="o_Message_header ml-2">
-                                <t t-if="messageView.message.author">
-                                    <div class="o_Message_authorName o_Message_authorRedirect o_redirect text-truncate" role="link" tabindex="0" t-on-click="_onClickAuthorName" title="Open profile">
-                                        <t t-if="messageView.message.originThread">
-                                            <t t-esc="messageView.message.originThread.getMemberName(messageView.message.author)"/>
-                                        </t>
-                                        <t t-else="">
-                                            <t t-esc="messageView.message.author.nameOrDisplayName"/>
-                                        </t>
-                                    </div>
-                                </t>
-                                <t t-elif="messageView.message.guestAuthor">
-                                    <div class="o_Message_authorName text-truncate">
-                                        <t t-esc="messageView.message.guestAuthor.name"/>
-                                    </div>
-                                </t>
-                                <t t-elif="messageView.message.email_from">
-                                    <a class="o_Message_authorName text-truncate" t-attf-href="mailto:{{ messageView.message.email_from }}?subject=Re: {{ messageView.message.subject ? messageView.message.subject : '' }}">
-                                        <t t-esc="messageView.message.email_from"/>
-                                    </a>
-                                </t>
-                                <t t-else="">
-                                    <div class="o_Message_authorName text-truncate">
-                                        Anonymous
-                                    </div>
-                                </t>
+                                <div t-att-class="threadView and threadView.__values.threadViewer.chatWindow and messageView.message.notifications.length > 0 ? 'w-100':''">
```

Comment:
It would be more readable with `t-att-class` object syntax:
```xml
<div t-att-class="{
   'w-100': threadView?.threadViewer.chatWindow and messageView.message.notifications.length > 0,
}"/>
```